ADAL/src/ADRequestParameters.h

```diff
@@ -46,7 +47,7 @@
 @property (retain, nonatomic) NSString* redirectUri;
 @property (retain, nonatomic) NSString* scope;
 @property (retain, nonatomic) ADUserIdentifier* identifier;
-@property (retain, nonatomic) ADTokenCacheAccessor* tokenCache;
+//@property (retain, nonatomic) ADTokenCacheAccessor* tokenCache;
```

Comment:
After going through the changes, I'm not sure I see a clear benefit of passing token cache around vs passing it in the request params. There's also an additional risk of forgetting to set cache when creating an object. What was the problem with storing it in request params?